csrc/scheduler/mma_utils.cpp

```diff
@@ -705,9 +705,37 @@ std::unordered_set<IterDomain*> getMmaDomainSet(
 
 } // namespace
 
+bool isConsumerAllocationInnerIDProducerAllocationInnerID(
+    const TensorView* producer,
+    const TensorView* consumer) {
+  // Get all the IDs from the innermost ID of the allocation domain of
+  // the consumer to the root domain of the consumer.
+  const auto vals = DependencyCheck::getAllValsBetween(
+      {consumer->getRootDomain().begin(), consumer->getRootDomain().end()},
+      {consumer->getMaybeAllocationDomain().back()});
+
+  const auto ids = ir_utils::filterByType<IterDomain>(vals);
+  const auto idsOnPath = std::vector<IterDomain*>(ids.begin(), ids.end());
```

Comment:
nit: According to https://google.github.io/styleguide/cppguide.html#Variable_Names, variable names should be in `snake_case`.